draft-ietf-quic-recovery.md

```diff
@@ -1560,6 +1566,7 @@ window.
      if (!InCongestionRecovery(sent_time)):
        congestion_recovery_start_time = now()
        congestion_window *= kLossReductionFactor
+       bytes_acked *= kLossReductionFactor
```

Comment:
I thought this would be set to 0 here.  Why reduce it?

Reply:
The current cwnd is (cwnd + bytes_acked/cwnd). Multiplying that by kLossReductionFactor means you multiply bytes_acked by kLossReductionFactor as well.

Reply:
Just thinking again... this is when a recovery episode starts and we enter congestion avoidance by ssthresh=cwnd. And bytes_acked is only used in congestion avoidance and doing nothing during slow start.

I think we simply clear bytes_acked to 0 as @ianswett said because it's a new start of congestion avoidance, or no need to touch bytes_acked because even if there is a previous leftover bytes_acked, it will not impact much.

Reply:
Thanks for the explanation.  I think I'm fine either way, but questions like this are one of the reason I liked the old approach of assuming it's the implementation's responsibility to deal with rounding/etc.

Reading this again, I realize there's a somewhat substantive difference in behavior between the previous pseudocode and this PR, because the previous pseudocode would increment CWND in bytes, not only max_datagram_size.

Reply:
This isn't a big difference, so I'm fine with any resolution here. @ianswett : I don't understand what you mean by the substantive difference. The previous code incremented by (bytes_acked/cwnd) fraction of max_datagram_size on each ack, this one accumulates that fraction until it is 1 and then adds max_datagram_size. It is functionally the same.

Reply:
I agree that this is a small difference either way.